Run7/step01/flats.cfg

```diff
@@ -0,0 +1,65 @@
+[ACQUIRE]
+flat
+
+[FLAT]
+SHUTTER = OPEN  # Shutter opens at start of acqusition, closes at end
+BCOUNT=1        # number of bias frames per flat image
+WL    =red    # wavelength filter to use for the flats
+HILIM =15.0    # maximum seconds for a flat field exposure
+LOLIM =0        # minimum seconds for a flat field exposure
+FILTERCONFIG = /gpfs/slac/lsst/fs2/u1/devel/ccs/bot-eotest-configs/Run6/calib.cfg
+
+
+
+#
+# Below has pairs from 100-200,000 e/pixel, log spaced with
```

Comment:
Just following up on a DM.  The comments in these lines are out of date.  The Neutral Density (ND) filters were used only through Run 5.  In that era we overlapped the flux ranges of the flat images obtained with different ND filters.  The list of fluxes starting at line 19 still contains duplications (e.g., 1357 e/pixel), which are no longer needed.  Also, the flux range extends up to 310000 e/pixel, which is way beyond full well.  In the Run 5 era we may have done this because the flat illumination was not particularly flat and we wanted to be sure to reach full well at the edge of the focal plane.  If the illumination is now more uniform (do we know?), something like 200000 might be sufficient as an upper limit

Reply:
Sean, I think we still want to keep the random ordering in flux for the flat pair acquisitions.  The original motivation for that was to mitigate systematic correlations with exposure time in the ptc and linearity analyses.